lib/uvobot/notifications/discourse_notifier.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './notifier'
+require_relative '../uvo_scraper'
+
+module Uvobot
+  module Notifications
+    class DiscourseNotifier < Notifier
+      def initialize(discourse_client, category = 'Štátne projekty', scraper = Uvobot::UvoScraper.new)
```

Comment:
Tu by som v klude tie default parametre dal prec a pleskol ich do top-levelu. Nech to je explicitne.